exec.go

```diff
@@ -114,7 +114,7 @@ func setPrimaryKey(rv reflect.Value, id int64) {
 	case reflect.Uint64:
 		rv.SetUint(uint64(id))
 	default:
-		err := fmt.Errorf("Unknown type to set primary key: %s", rv.Type())
+		err := fmt.Errorf("unknown type to set primary key: %s", rv.Type())
```

Comment:
I that a rule (good practise) I dont know about? Starting error messages with small case?

Reply:
Yep